app/assets/javascripts/jquery.countdown.js

```diff
@@ -0,0 +1,206 @@
+/*!
```

Comment:
This file should probably be in vendor/assets/javascripts and explicitly required in application.js